tutor/src/components/error-monitoring/async-load-error.jsx

```diff
@@ -1,21 +1,15 @@
 import React from 'react';
 import { Button } from 'react-bootstrap';
 import OXColoredStripe from 'shared/components/ox-colored-stripe';
-import { isReloaded, reloadOnce, forceReload } from '../../helpers/reload';
+import { forceReload } from '../../helpers/reload';
 
 export default class AsyncLoadError extends React.PureComponent {
 
   static propTypes = {
     error: React.PropTypes.object.isRequired,
   }
 
-  componentWillMount() {
-    reloadOnce();
-  }
-
   render() {
-    if (!isReloaded()) { return null; }
-
     return (
```

Comment:
Hm, I'm pretty sure we still want to reload on error.  Why remove this?

Reply:
oh i thought we talked about this?  i guess i can across this because when there's an error with the scores report modal saving, the error modal pops up and then the whole page reloads without the user closing or being able to read the modal and it's very disorienting

Reply:
oh, sorry! This is a different type of error.  This error is caused by a JS chunk failing to load, most likely because we deployed and the file that it needs is no longer in the directory.  If we reload then the browser will get the update build and all will be well.

it's a different modal that reloads in response to server errors, it's in https://github.com/openstax/tutor-js/blob/master/tutor/src/components/error-monitoring/index.cjsx

That's the one that I think we can get rid of.

Reply:
ah my bad, will fix these